client/src/services/apiProxy.ts

```diff
@@ -59,7 +62,22 @@ async function requestInner(path: string, params?: any, autoLogoutOnAuthError =
 }
 
 export async function request(path: string, params?: any, autoLogoutOnAuthError = true) {
```

Comment:
I think it might be a bit clear to keep all of the Protobuf logic inside of `requestProto`. Maybe something like this...

```
export async function request(path: string, params?: any, autoLogoutOnAuthError = true) {
    if (isProtoEligible(path) && isProtoEnabled()) {
        return requestProto(path, params, autoLogoutOnAuthError);
    } else {
        return requestJson(path, params, autoLogoutOnAuthError)
    }
}

export async function requestJson(path: string, params?: any, autoLogoutOnAuthError = true) {
    const body = await requestInner(path, params, autoLogoutOnAuthError, false)
    return body.json();
}

export async function requestProto(path: string, params?: any, autoLogoutOnAuthError = true) {
    const body = await requestInner(path, params, autoLogoutOnAuthError, true)
    const buffer = body.arrayBuffer();
    const value = new Uint8Array(buffer);
    return parser(value);
}
```

Reply:
Yes, it's due to the package-lock.json. I need to update it to fix the issue. Good call here. I will refactor it.

Reply:
You can run this command to avoid using indeed nexus `npm install --registry=https://registry.npmjs.org/`